kvbc/test/sparse_merkle_storage/db_adapter_unit_test.cpp

```diff
@@ -1271,6 +1483,118 @@ TEST_P(db_adapter_custom_blockchain, delete_genesis_and_verify_stale_index) {
   ASSERT_FALSE(hasProvableStaleIndexKeysSince(adapter.getDb(), 2));
 }
 
+TEST_P(db_adapter_custom_blockchain, delete_genesis_and_verify_stale_key_deletion_with_non_provable_keys) {
+  const Sliver skey1{"key1"};
+  const Sliver skey2{"key2"};
+  concord::kvbc::v2MerkleTree::DBAdapter::NonProvableKeySet nonProvableKeySet = {skey1, skey2};
+  auto adapter = DBAdapter{GetParam()->newEmptyDb(), true, nonProvableKeySet};
+
+  const auto ver1StaleNonProvableKey1 = DBKeyManipulator::genNonProvableDbKey(1, skey1);
+  const auto ver1StaleNonProvableKey2 = DBKeyManipulator::genNonProvableDbKey(1, skey2);
```

Comment:
Nit: I think these are not used.

Reply:
It is weird that the compiler lets this code compile. I was sure we disabled unused variables.
Fixed.